packages/shared/src/components/profile/LanguageDropdown.tsx

```diff
@@ -22,7 +20,7 @@ type Props = {
 
 const defaultIcon = <LanguageIcon className="ml-0 mr-1" />;
 
-const LanguageDropdownDefault = ({
+export const LanguageDropdown = ({
```

Comment:
We should also see down the line (after some more adoption?) to make language required in registration flow on API? (not blocker for this PR)